resources/translations/en.edn

```diff
@@ -150,7 +150,7 @@
               :intro [:p "The catalogue contains all the resources that you can apply for in this system."]
               :more-info "More info"
               :continue-existing-application "Continue existing application"
-              :continue-existing-application-intro "You have existing unfinished applications that you can continue from here."
+              :continue-existing-application-intro [:div.intro [:p "You have existing unfinished applications that you can continue from here."]]
```

Comment:
I'd remove the redundant word "existing" (or "unfinished", but it conveys more information so I'd keep that).